lib/utils/GraphAPI.dart

```diff
@@ -23,24 +23,20 @@ class GraphAPI {
           variables: {"email": user.email, "password": user.password}),
     );
 
-      if (result.loading) {
-      return Text('Loading');
-    }
-
     if (result.hasException) {
       print(result.exception);
       Scaffold.of(context).showSnackBar(SnackBar(
           content: Text(result.exception.toString(),
```

Comment:
Figure this is just a placeholder but would be good to implement some more user friendly errors here

Reply:
login also takes around 3secs. the snack bar is shown then you are brought to the homepage. The only issue I noticed on my end has to do with the login progress bar. We can look at this in a quick call also.